bitnami/contour/README.md

```diff
@@ -52,6 +52,24 @@ Bitnami charts allow setting resource requests and limits for all containers ins
 
 To make this process easier, the chart contains the `resourcesPreset` values, which automatically sets the `resources` section according to different presets. Check these presets in [the bitnami/common chart](https://github.com/bitnami/charts/blob/main/bitnami/common/templates/_resources.tpl#L15). However, in production workloads using `resourcePreset` is discouraged as it may not fully adapt to your specific needs. Find more information on container resource management in the [official Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/).
 
+### Prometheus metrics
+
+This chart can be integrated with Prometheus by setting `*.metrics.enabled` (under the `dataPlane` and `controlPlane` sections) to true. This will expose the Contour and Envoy native Prometheus ports in the containers. Additionally, it will deploy several `metrics` services. These `metrics` services will have the necessary annotations to be automatically scraped by Prometheus.
```

Comment:
Copy-paste error, there are no `dataPlane/controlPlane` in `countour`
Contour controls metrics with the `metrics.serviceMonitor.enabled` I think, Envoy uses the same value but requires `envoy.service.exposeMetrics` additionally.

Reply:
Good catch!